src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioTransformationServiceImpl.java

```diff
@@ -298,11 +304,17 @@ private Map<MediaEntity, Path> downloadAndSaveMediaToWorkspace(List<MediaEntity>
     @Override
     public Path saveMediaToWorkspace(MediaEntity mediaEntity) throws IOException {
         UUID id = getMediaLocation(mediaEntity).orElseThrow(
```

Comment:
as is we'd be using the UUID of the file in unstructured. Would the file in DETS be guaranteed to have the same UUID? Or we should update `getMediaLocation` to find the unique UUID for the file in DETS?

Reply:
what I was wondering if it's possible that the same file has got 2 different UUIDs for DETs and unstructured

Reply:
possibly - but i think that would be an error with data upload/storage - since the eod only has 1 uuid